frontend/src/javascript/utils/geometryUtils.js

```diff
@@ -0,0 +1,33 @@
+export function rescalePoints(points) {
+    const sceneBoundingBoxSize = 400;
+
+    let minX = points
```

Comment:
Spróbuj może `let minX = points.map(point => point.x).getMinValue()` to wtedy chyba powinno się zmieścić w jednej linii. Teraz przez to automatyczne formatowanie jest to stranie rozwleczone na wiele linii.

Reply:
Zrobione